app/script_variant.py

```diff
@@ -29,6 +29,14 @@ def read_dictionary(file_name):
 JAPANESE_LOCATION_DICTIONARY = read_dictionary('jp_location_dict.txt')
 
 
+def is_kanji(word):
+    """
+    Returns word is kanji or not.
+    [\u3400-\u9fff]: kanji
+    """
+    return re.match(ur'([\u3400-\u9fff])', word)
```

Comment:
This range is actually not very accurate. Some of kanji characters are out of this range. Can you add TODO to update this to contains all kanji characters?